tdrs-backend/tdpservice/search_indexes/migrations/0007_ssp_m1_ssp_m2_ssp_m3.py

```diff
@@ -87,7 +87,7 @@ class Migration(migrations.Migration):
                 ('RELATIONSHIP_HOH', models.IntegerField(null=True)),
                 ('PARENT_MINOR_CHILD', models.IntegerField(null=True)),
                 ('NEEDS_PREGNANT_WOMAN', models.IntegerField(null=True)),
-                ('EDUCATION_LEVEL', models.IntegerField(null=True)),
+                ('EDUCATION_LEVEL', models.CharField(max_length=2, null=True)),
```

Comment:
i don't think we should modify existing migrations

Reply:
Oh, I didn't catch that. I will revert that and make sure the new migration captures that.

Reply:
Resolved